core/src/executor/context/blend_context.rs

```diff
@@ -73,8 +81,9 @@ impl<'a> BlendContext<'a> {
     pub fn get_alias_values(&self, alias: &str) -> Option<Vec<Value>> {
         if self.table_alias == alias {
             let values = match &self.row {
-                Some(Row(values)) => values.clone(),
-                None => self.columns.iter().map(|_| Value::Null).collect(),
+                BlendContextRow::Shared(row) => row.as_ref().0.clone(),
```

Comment:
```suggestion
                BlendContextRow::Shared(row) => row.0.clone(),
```

I don't think `as_ref` is necessary.